N24DCCN167/BaiTap_buoi8/6.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+#include <vector>
+using namespace std;
+
+int main()
+{
+    string str = "    Cau   lac bo   ITMC  Cua  Toi";
+    string newStr = "";
+    vector<string> res;
+    for (int i = 0; i < str.length(); i++)
+    {
+        if (str[i] != ' ')
+        {
+            newStr += str[i];
+        }
+        else
+        {
+            if (newStr != "")
+            {
+                res.push_back(newStr);
+                newStr = "";
+            }
+        }
+    }
+    if (str[str.length() - 1] != ' ')
+        res.push_back(newStr);
+    for (string i : res)
```

Comment:
In sai. Yêu cầu đề là output phải dạng ntn [“Cau”,“lac”, “bo”, “ITMC”]. Còn của em thì chỉ là từng từ sau đó xuống hàng. Sử lại nha